platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/State.java

```diff
@@ -244,7 +253,7 @@ public String getInfoString(final int hashDepth) {
     @Override
     public String toString() {
         return new ToStringBuilder(this)
-                .append("platformState", getPlatformState())
+                .append("platformState", getWritablePlatformState())
```

Comment:
Why is a `toString` method accessing _writable_ state?  Should this be a `readable` state?

Reply:
It's mostly because `getReadablePlatformState` throws an exception in this particular class. However, I agree - it looks confusing, so I changed implementation of `getReadablePlatformState`. Now it has exactly the same implementation as `getWritablePlatformState`

Reply:
Fixed